app/Http/Controllers/UserController.php

```diff
@@ -150,4 +153,24 @@ public function cancelAccountDeletion(Request $request): JsonResponse
 
         return response()->json(['success' => true]);
     }
+
+    /**
+     * Avatars unfortunately are required to use stable usernames due to downstream concerns such as emulators.
+     * This makes them difficult to cache. There isn't anything that inherently can cachebust them like a query parameter.
+     * We'll manually tell Cloudflare when it's time to cachebust these assets.
+     */
+    private function purgeUserAvatarFromCloudflare(User $user): void
+    {
+        $token = config('services.cloudflare.purge_token');
+        $zone = config('services.cloudflare.zone');
+        $mediaUrl = config('filesystems.disks.media.url');
+
+        if ($token && $zone && $mediaUrl) {
+            Http::withToken($token)
+                ->post("https://api.cloudflare.com/client/v4/zones/{$zone}/purge_cache", [
+                    // "https://media.retroachievements.org/UserPic/Scott.png"
+                    'files' => ["{$mediaUrl}/UserPic/{$user->username}.png"],
+                ]);
```

Comment:
Do we want to try to capture when this errors? Not sure what types of errors it returns, or if the URL might change, or if there's just a generic network error.